src/components/Filter.jsx

```diff
@@ -0,0 +1,171 @@
+import React, { useState, useEffect } from 'react'
+import axios from 'axios'
+import { FaFilter } from 'react-icons/fa'
+
+import '../styles/Filter.css'
+
+const Filter = ({mnag, setManga}) => {
+  const [isChecked, setIsChecked] = useState({
+    lowtohigh: false,
+    hightolow: false,
+  })
+
+  const [categories, setCategories] = useState({
+    action: false,
+    romance: false,
+    sliceOfLife: false,
+    fantasy: false,
+    comedy: false,
+    adventure: false,
+    sciFi: false
+  })
+
+  const radioHandler = e => {
+    const { name } = e.target;
+    if (name === 'low') setIsChecked({ ...isChecked, lowtohigh: true, hightolow: false })
+    else setIsChecked({ ...isChecked, lowtohigh: false, hightolow: true })
+  }
+
+  const checkboxHandler = (e) => {
+    const { name } = e.target;
+    if (name === "action") {
+      setCategories({ ...categories, action: categories.action === true ? false : true })
+    } else if (name === "romance") {
+      setCategories({ ...categories, romance: categories.romance === true ? false : true })
+    } else if (name === "sliceoflife") {
+      setCategories({ ...categories, sliceOfLife: categories.sliceOfLife === true ? false : true })
+    } else if (name === "fantasy") {
+      setCategories({ ...categories, fantasy: categories.fantasy === true ? false : true })
+    } else if (name === "comedy") {
+      setCategories({ ...categories, comedy: categories.comedy === true ? false : true })
+    } else if (name === "adventure") {
+      setCategories({ ...categories, adventure: categories.adventure === true ? false : true })
+    } else {
+      setCategories({ ...categories, sciFi: categories.sciFi === true ? false : true })
+    }
+  }
+
+  useEffect(() => {
+    (async () => {
+      const data = await axios.get('/api/products');
+      let items = data.data.products
+      setManga([...items])
+    })()
+  }, [])
+
+  return (
+    <section className="filter-container">
+      <div className='filter'>
+        <div className="filter-header">
+          <h2><FaFilter /><span>Filters</span></h2>
+          <button>Clear All</button>
+        </div>
+        <div className="filter-sort">
+          <span>SORT</span>
+          <div>
+            <input
+              type="radio"
+              name='low'
+              value='low'
+              checked={isChecked.lowtohigh}
+              onChange={radioHandler}
+            />
+            <label>Low to High</label><br />
```

Comment:
In the label, you can use htmlFor attribute so when we click on label the corresponding radio input will be checked